ntbs-service/Pages/Notifications/Edit/Episode.cshtml.cs

```diff
@@ -51,6 +41,11 @@ public override async Task<IActionResult> OnGetAsync(int id, bool isBeingSubmitt
                                         nameof(Hospital.HospitalId), 
                                         nameof(Hospital.Name));
 
+            if (Episode.ShouldValidateFull)

```

Comment:
Just changed the order here to make all the page models more consistent - validation happens last